server/src/test/scala/org/http4s/server/middleware/DefaultHeadSpec.scala

```diff
@@ -12,10 +12,10 @@ class DefaultHeadSpec extends Http4sSpec {
       Ok("hello")
 
     case GET -> Root / "special" =>
-      Ok().putHeaders(Header("X-Handled-By", "GET"))
+      Ok(Header("X-Handled-By", "GET"))
```

Comment:
This is a little weird, because the first argument isn't the body.  Maybe we should force `Ok((), Header(..)` in this case?